### ISSUE TITLE
Can I get your data split? and when will the evaluate.py be released?

### ISSUE THREAD

**Issue by Bosen-Zhang** [THU-BPM/SelfORE#7]
Hello, thanks for your contribution.
Can I get these data and code?
Best wishes!

**Comment by vhientran** [comment]
Hi @usoonxu , Could you please release the script of the file evaluate.py? Thanks so much!

**Comment by xuminghu** [comment]
Thanks for your interest in our work.
   For the evaluate.py, we directly use the B-cubed, V-measure, and ARI in the sklearn library as evaluation metrics. Since it is an unsupervised clustering task, we do not need label alignment and we could use above metrics directly. We will update evaluate.py recently to facilitate the evaluation results.
   For the dataset, we provided a [link](https://hadyelsahar.github.io/t-rex/) to obtain the T-ReX dataset. We do not have the copyright of the NYT-FB dataset, so please contact Diego Marcheggiani to obtain it.

**Comment by xuminghu** [comment]
For data split, we followed the [previous work](https://www.aclweb.org/anthology/P19-1133/). Since it is an unsupervised task, we randomly selected 80% of the data as the training set and 20% of the data as the validation set.

**Comment by vhientran** [comment]
Thank you so much!
